pkg/kademlia/kademlia.go

```diff
@@ -260,6 +262,7 @@ func GetIntroNode(addr string) (*pb.Node, error) {
 			Transport: defaultTransport,
 			Address:   addr,
 		},
+		Type: pb.NodeType_ADMIN,
```

Comment:
This is the bootstrap node, I could change the nodetype name to reflect this

Reply:
This probably should be `pb.NodeType_Unknown` or similar.